### ISSUE TITLE
can i use "Deepl"?

### ISSUE THREAD

**Issue by jos582** [NyashniyVladya/Translator3000#25]
thank you for creating this wonderful program.
so far it is the best I have found to translate renpy games, but I would like to know if it is possible to use another translator besides google, like "Deepl", since it is the best quality one I know.

**Comment by DUB1401** [comment]
Да, было бы весьма неплохо добавить Deepl. Качество куда лучше. Присоединяюсь.

**Comment by Ilkhan16** [comment]
+1
